osf_tests/factories.py

```diff
@@ -485,27 +493,29 @@ class Meta:
 
     @classmethod
     def _create(cls, *args, **kwargs):
-        branched_from = kwargs.get('branched_from')
-        initiator = kwargs.get('initiator')
+        title = kwargs.pop('title', None)
+        description = kwargs.pop('description', None)
+        branched_from = kwargs.get('branched_from', None)
         registration_schema = kwargs.get('registration_schema')
         registration_metadata = kwargs.get('registration_metadata')
         provider = kwargs.get('provider')
-        if not branched_from:
-            project_params = {}
-            if initiator:
-                project_params['creator'] = initiator
-            branched_from = ProjectFactory(**project_params)
-        initiator = branched_from.creator
+        initiator = branched_from.creator if branched_from else kwargs.get('initiator', None)
+        initiator = initiator or kwargs.get('user', None) or kwargs.get('creator', None) or UserFactory()
         registration_schema = registration_schema or models.RegistrationSchema.objects.first()
         registration_metadata = registration_metadata or {}
         provider = provider or models.RegistrationProvider.objects.first() or RegistrationProviderFactory(_id='osf')
         draft = models.DraftRegistration.create_from_node(
-            branched_from,
+            node=branched_from,
```

Comment:
For factory, no longer creating project automatically, DraftRegistrationFactory either uses the `branched_from` passed in, or creates a DraftNode as part of the `create_from_node` method.